### PR TITLE
add FirestormPrism

### PR DESCRIPTION
**Pull request by medaey**
Add the **FirestormPrism** for **repeater** weapons

image

### CODE REVIEW

File: data/parts/repeater/prisms/FirestormPrism.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: FirestormPrism
```
Comment (atomicptr) [inline data/parts/repeater/prisms/FirestormPrism.yml]: This should probably be "Firestorm Prism". Don't forget to reset/rebuild the map :).

**Review by atomicptr — CHANGES_REQUESTED** [review]
I suppose the yarn.lock changes are just an update? Beyond the typo in the name, it looks good tyvm :)

File: data/parts/repeater/prisms/FirestormPrism.yml (hunk 1 above)
Comment (medaey) [reply data/parts/repeater/prisms/FirestormPrism.yml]: Oups my bad, it's okay or is it better that I merge my two commit into one commit ?
And yes for **yarn.lock** it's just an update ^^

[64dfbd9](https://github.com/atomicptr/dauntless-builder/pull/150/commits/64dfbd9dfe9544bebe0af2f42b6852bed0072980)

**Review by atomicptr — APPROVED** [review]

**Comment by atomicptr** [comment]
Looks good, tyvm

File: data/parts/repeater/prisms/FirestormPrism.yml (hunk 1 above)
Comment (atomicptr) [reply data/parts/repeater/prisms/FirestormPrism.yml]: Yeah I'll just squash the merge anyway :)

**Pull request merged**